src/Debug/WrappedDelayedListener.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Biig\Component\Domain\Debug;
+
+use Symfony\Component\VarDumper\Caster\ClassStub;
+
+class WrappedDelayedListener
+{
+    /**
+     * @var callable
+     */
+    private $listener;
+
+    /**
+     * @var string
+     */
+    private $pretty;
+
+    /**
+     * @var string Class function stub
+     */
+    private $stub;
+
+    /**
+     * @var mixed|string
+     */
+    private $name;
+
+    private static $hasClassStub;
```

Comment:
No comment, and why `static` ?

Reply:
Comment added. It is static because you only to test once if ClassStub class exists